packages/core/README.md

```diff
@@ -82,9 +82,11 @@ An array of Chains that your app supports:
 type Chain = {
   id: ChainId // hex encoded string, eg '0x1' for Ethereum Mainnet
   namespace?: 'evm' // string indicating chain namespace. Defaults to 'evm' but will allow other chain namespaces in the future
-  rpcUrl: string // used for network requests (eg Alchemy or Infura end point)
-  label: string // used for display, eg Ethereum Mainnet
-  token: TokenSymbol // the native token symbol, eg ETH, BNB, MATIC
+  // PLEASE NOTE: Some wallets require an rpcUrl, label, and token for actions such as adding a new chain. 
+  // It is recommended to include rpcUrl, label, and token for full functionality. 
+  rpcUrl?: string // Recommended to include. Used for network requests (eg Alchemy or Infura end point).
```

Comment:
It would be helpful to add all these prop notes as TS notes within `packages/common/src/types.ts` 
Def will enhance devEx while we are working in this area